src/common/merge.ts

```diff
@@ -143,8 +194,12 @@ export const tryMerge = async (
   } else if (checkPullRequestTitleForMergePreset(pullRequestTitle) === false) {
     logInfo(`Pull request version bump is not allowed by PRESET.`);
   } else if (
-    isMatch(commitAuthorName, allowedAuthorName) === false &&
-    disabledForManualChanges === true
+    enabledForManualChanges === false &&
+    (await getIsModified(octokit, {
+      pullRequestNumber,
+      repositoryName,
+      repositoryOwner,
+    })) === true
```

Comment:
Do you think we could assign it to a constant instead of inlining to improve readability?

Reply:
We can, but it will make execution less efficient. Wdyt?

Reply:
We can still evaluate it lazily if it's in the else block. I didn't mean to put it before this chain.